CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - `[@jest/core]` [**BREAKING**] Group together open handles with the same stack trace ([#13417](https://github.com/jestjs/jest/pull/13417), & [#14543](https://github.com/jestjs/jest/pull/14543))
 - `[@jest/core]` Add `perfStats` to surface test setup overhead ([#14622](https://github.com/jestjs/jest/pull/14622))
 - `[@jest/core]` [**BREAKING**] Changed `--filter` to accept an object with shape `{ filtered: Array<string> }` to match [documentation](https://jestjs.io/docs/cli#--filterfile) ([#13319](https://github.com/jestjs/jest/pull/13319))
+- `[@jest/core]` [**BREAKING**] Revert breaking changes for `detectOpenHandles`. ([#14789](https://github.com/jestjs/jest/pull/14789))
```

Comment:
should this entry be grouped with the line on line 10?

Reply:
Yes, this looks good to me.
Also the original second PR on line 10 seemed to be irrelevant, so I replaced it.

Reply:
We might also be able to remove `[**BREAKING**]`.